ext/node/polyfills/_process/streams.mjs

```diff
@@ -47,7 +48,7 @@ export function createWritableStdioStream(writer, name, warmup = false) {
       cb(err);
       this._undestroy();
       if (!this._writableState.emitClose) {
-        nextTick(() => this.emit("close"));
+        //  nextTick(() => this.emit("close"));
```

Comment:
Is this on purpose?

Reply:
Added this line back and removed explicit `writer.close` in the event handler instead. This is needed because stdio streams can be un-destroyed using `stdout.undestroy()`, so we never really want to close the resource (until cleanup)